setup.cfg

```diff
@@ -0,0 +1,35 @@
+# FILEPATH: setup.cfg
+
+[metadata]
+name = prompt_systematic_review
+version = 0.1.0
```

Comment:
why not 0.0.1?

Reply:
that was the default, but we can change that

Reply:
all good, leave as default, was curious